c3-angular.js

```diff
@@ -51,9 +59,16 @@ var counter = Math.ceil((Math.random() * 1000));
               size: size,
               padding: padding,
               grid: grid,
-              legend: {
-                show: showLegend
-              }
+              legend: scope.legend || {},
+              point: scope.point || {},
+              regions: scope.regions || {},
+              tooltip: scope.tooltip || {},
+              line: scope.line || {},
+              area: scope.area || {},
+              bar: scope.bar || {},
+              pie: scope.pie || {},
```

Comment:
Ano. Už přidávalo legend a point, tak než aby se to muselo zas někdy upravovat, tak jsem to přidal všechno.